contrib/win32/openssh/win32iocompat.vcxproj

```diff
@@ -202,6 +205,9 @@
       <ControlFlowGuard>Guard</ControlFlowGuard>
       <AdditionalOptions>/Gy %(AdditionalOptions)</AdditionalOptions>
     </ClCompile>
+	<Lib>
+      <AdditionalDependencies>Wtsapi32.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
Windows inbox builds are onecore builds.

Please check if can be compiled with onecore lib